scripts/travis/build.sh

```diff
@@ -29,7 +30,23 @@ else
     -DUSE_SYSTEM_PCRE=OFF \
     -DUSE_SYSTEM_ZLIB=ON \
     ..
-  make -j2
+  else
+    # osx
+    export CC=clang
+    export CXX=clang++
+    export CMAKE_PREFIX_PATH=/usr/local/Cellar/qt/5.11.1/lib/cmake
```

Comment:
Does this work as expected? The prefix path would normally reference the install prefix, which I would expect to be /usr/local/Cellar/qt/5.11.1, the Qt5_DIR has to go deeper into lib usually. No objections if it works as expected.

Reply:
Yup, it builds: 
https://travis-ci.org/OpenChemistry/avogadrolibs/jobs/429095892

Reply:
Ideally, the Travis builds would be useful for AppImage and Mac continuous / nightly binary packages.

Reply:
I saw it built, I just wondered if the variable was being used/affected it - poorly worded question. It isn't how it is intended to be used, and so I was surprised it would work, but CMake can surprise me at times.